contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,571 @@
+# Local Raw Block Consumption via Persistent Volume Source
```

Comment:
I am fine with it.  The changes proposed here can be applied to any volume plugin, not just local storage.